### ISSUE TITLE
NExt Oneways

### ISSUE THREAD

**Issue by tommargar** [Killface1980/RoadsUnited_Core#7]
I have a similar error and arrows and lines have disappeared from elevated NEXT oneways. On other road types lane markings seem to work but not the asphalt texture (brightness control doesn't work and the texture looks like it is the vanilla one to me)
